lib/pbench/server/api/__init__.py

```diff
@@ -21,6 +22,7 @@
 ALLOWED_EXTENSIONS = {"xz"}
 
 app = None
+_MAX_UPLOAD_DATA_SIZE = 2 ** 30
```

Comment:
Can we consider making this a configuration parameter instead of a hard-coded value?

Reply:
Just noticed we already have `rest_max_content_length = 100 * 1024 * 1024` in our server config file. I will just use that (its approx 1 GB)